app/controllers/address_info_controller.rb

```diff
@@ -0,0 +1,7 @@
+class AddressInfoController < ApplicationController
```

Comment:
@AkshayPashnathSuresh 
naming should be plural again. 
`AddressInfosController`